.gitignore

```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+src/tailwind.css
```

Comment:
it looks like this gets created dynamically every time and is not needed in src code just at runtime